cowprotocol/solver_dashboard/number_capped_reward_batches_3670624.sql

```diff
@@ -0,0 +1,33 @@
+-- This query finds the number of transactions where the rewards reached the upper bound
+
+with
+batch_rewards as (
+    select
+        winning_solver,
+        tx_hash,
+        reward,
+        date_add('day', 1, date_trunc('week', date_add('day', -1, time))) as week_start
+    from query_2777544 as bs
```

Comment:
I believe the query you reference here is not part of this repo. If that is indeed the case, then we should migrate 2777544 as well in this repo and under the cowprotocol namespace

Reply:
This query is linked to internal monitoring. Will include the queries in a subfolder